ship-as-trial.md

```diff
@@ -56,7 +62,19 @@ Similarly, if you need to know in the browser process whether a feature should b
 
 ### Testing
 If you want to test your code's interactions with the framework, you'll need to generate some tokens of your own. To generate your own tokens, use [/tools/origin_trials/generate_token.py](https://code.google.com/p/chromium/codesearch#chromium/src/tools/origin_trials/generate_token.py).
-You can generate signed tokens for localhost, or for 127.0.0.1, or for any origin that you need to help you test.
+You can generate signed tokens for localhost, or for 127.0.0.1, or for any origin that you need to help you test. For example:
+
+```
+tools/origin_trials/generate_token.py --key-file=tools/origin_trials/eftest.key http://localhost:8000 MyFeature
+```
+
+The `eftest.key` file is the private key for the test keypair used by Origin Trials unit tests (tokens generated with this key will **not** work in the browser by default; see the [Developer Guide](developer-guide.md) for instructions on creating real tokens). To use a test token with the browser, run Chrome with the command-line flag:
```

Comment:
This could/should have probably mentioned that this key is the key used by content_shell (and thus by layout tests), and tokens generated by this key will by default work in there.